From: Michael Biebl <biebl@debian.org>
Date: Mon, 8 Sep 2014 12:56:41 +0200
Subject: Fix paths in man pages

In Debian we still use the split-/usr setup, where systemd and udev are
installed into / and not /usr.
Fix up the man pages to point at the correct locations.

LP: #1357782
Closes: #717491
---
 man/bootchart.conf.xml                       |   6 +-
 man/coredump.conf.xml                        |   2 +-
 man/hwdb.xml                                 |   8 +-
 man/journald.conf.xml                        |   2 +-
 man/logind.conf.xml                          |   2 +-
 man/resolved.conf.xml                        |   2 +-
 man/sysctl.d.xml                             |   2 +-
 man/systemctl.xml                            |   4 +-
 man/systemd-activate.xml                     |   6 +-
 man/systemd-backlight@.service.xml           |   2 +-
 man/systemd-binfmt.service.xml               |   2 +-
 man/systemd-bootchart.xml                    |   6 +-
 man/systemd-bus-proxyd.xml                   |   2 +-
 man/systemd-coredump.xml                     |   2 +-
 man/systemd-cryptsetup-generator.xml         |   2 +-
 man/systemd-cryptsetup@.service.xml          |   2 +-
 man/systemd-debug-generator.xml              |   2 +-
 man/systemd-efi-boot-generator.xml           |   2 +-
 man/systemd-fsck@.service.xml                |   3 +-
 man/systemd-fstab-generator.xml              |   2 +-
 man/systemd-getty-generator.xml              |   2 +-
 man/systemd-gpt-auto-generator.xml           |   2 +-
 man/systemd-halt.service.xml                 |   6 +-
 man/systemd-hibernate-resume-generator.xml   |   2 +-
 man/systemd-hibernate-resume@.service.xml    |   2 +-
 man/systemd-hostnamed.service.xml            |   2 +-
 man/systemd-hwdb.xml                         |   2 +-
 man/systemd-initctl.service.xml              |   2 +-
 man/systemd-journal-gatewayd.service.xml     |   2 +-
 man/systemd-journald.service.xml             |   2 +-
 man/systemd-localed.service.xml              |   2 +-
 man/systemd-logind.service.xml               |   2 +-
 man/systemd-machine-id-commit.service.xml    |   2 +-
 man/systemd-machined.service.xml             |   2 +-
 man/systemd-modules-load.service.xml         |   2 +-
 man/systemd-networkd-wait-online.service.xml |   2 +-
 man/systemd-networkd.service.xml             |   4 +-
 man/systemd-quotacheck.service.xml           |   2 +-
 man/systemd-random-seed.service.xml          |   2 +-
 man/systemd-remount-fs.service.xml           |   2 +-
 man/systemd-resolved.service.xml             |   2 +-
 man/systemd-rfkill@.service.xml              |   2 +-
 man/systemd-shutdownd.service.xml            |   2 +-
 man/systemd-sleep.conf.xml                   |   2 +-
 man/systemd-socket-proxyd.xml                |   4 +-
 man/systemd-suspend.service.xml              |   6 +-
 man/systemd-sysctl.service.xml               |   2 +-
 man/systemd-system-update-generator.xml      |   2 +-
 man/systemd-system.conf.xml                  |   4 +-
 man/systemd-sysv-generator.xml               |   2 +-
 man/systemd-timedated.service.xml            |   2 +-
 man/systemd-timesyncd.service.xml            |   3 +-
 man/systemd-udevd.service.xml                |   2 +-
 man/systemd-update-done.service.xml          |   2 +-
 man/systemd-update-utmp.service.xml          |   2 +-
 man/systemd-user-sessions.service.xml        |   3 +-
 man/systemd-vconsole-setup.service.xml       |   2 +-
 man/systemd.directives.xml                   | 140 +++++++++++++--------------
 man/systemd.link.xml                         |   6 +-
 man/systemd.netdev.xml                       |   6 +-
 man/systemd.network.xml                      |   6 +-
 man/systemd.preset.xml                       |  14 +--
 man/systemd.unit.xml                         |  10 +-
 man/systemd.xml                              |   3 +-
 man/timedatectl.xml                          |   2 +-
 man/timesyncd.conf.xml                       |   2 +-
 man/udev.xml                                 |   8 +-
 67 files changed, 172 insertions(+), 176 deletions(-)

diff --git a/man/bootchart.conf.xml b/man/bootchart.conf.xml
index bf6ca0b..84316bd 100644
--- a/man/bootchart.conf.xml
+++ b/man/bootchart.conf.xml
@@ -55,7 +55,7 @@
     <para><filename>/etc/systemd/bootchart.conf</filename></para>
     <para><filename>/etc/systemd/bootchart.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/bootchart.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/bootchart.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/bootchart.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -122,10 +122,10 @@
         <term><varname>Init=[path]</varname></term>
         <listitem><para>Configures bootchart to run a non-standard
         binary instead of
-        <filename>/usr/lib/systemd/systemd</filename>. This option is
+        <filename>/lib/systemd/systemd</filename>. This option is
         only relevant if bootchart was invoked from the kernel command
         line with
-        init=/usr/lib/systemd/systemd-bootchart.</para></listitem>
+        init=/lib/systemd/systemd-bootchart.</para></listitem>
       </varlistentry>
 
       <varlistentry>
diff --git a/man/coredump.conf.xml b/man/coredump.conf.xml
index 0b7329b..795867d 100644
--- a/man/coredump.conf.xml
+++ b/man/coredump.conf.xml
@@ -52,7 +52,7 @@
     <para><filename>/etc/systemd/coredump.conf</filename></para>
     <para><filename>/etc/systemd/coredump.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/coredump.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/coredump.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/coredump.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/hwdb.xml b/man/hwdb.xml
index b4460c1..e29824e 100644
--- a/man/hwdb.xml
+++ b/man/hwdb.xml
@@ -40,17 +40,17 @@
 
   <refsect1><title>Hardware Database Files</title>
       <para>The hwdb files are read from the files located in the
-      system hwdb directory <filename>/usr/lib/udev/hwdb.d</filename>,
+      system hwdb directory <filename>/lib/udev/hwdb.d</filename>,
       the volatile runtime directory <filename>/run/udev/hwdb.d</filename>
       and the local administration directory <filename>/etc/udev/hwdb.d</filename>.
       All hwdb files are collectively sorted and processed in lexical order,
       regardless of the directories in which they live. However, files with
       identical filenames replace each other. Files in <filename>/etc</filename>
       have the highest priority, files in <filename>/run</filename> take precedence
-      over files with the same name in <filename>/usr/lib</filename>. This can be
+      over files with the same name in <filename>/lib</filename>. This can be
       used to override a system-supplied hwdb file with a local file if needed;
       a symlink in <filename>/etc</filename> with the same name as a hwdb file in
-      <filename>/usr/lib</filename>, pointing to <filename>/dev/null</filename>,
+      <filename>/lib</filename>, pointing to <filename>/dev/null</filename>,
       disables the hwdb file entirely. hwdb files must have the extension
       <filename>.hwdb</filename>; other extensions are ignored.</para>
 
@@ -70,7 +70,7 @@
       <para>The content of all hwdb files is read by
       <citerefentry><refentrytitle>systemd-hwdb</refentrytitle><manvolnum>8</manvolnum></citerefentry>
       and compiled to a binary database located at <filename>/etc/udev/hwdb.bin</filename>,
-      or alternatively <filename>/usr/lib/udev/hwdb.bin</filename> if you want ship the compiled
+      or alternatively <filename>/lib/udev/hwdb.bin</filename> if you want ship the compiled
       database in an immutable image.
       During runtime only the binary database is used.</para>
   </refsect1>
diff --git a/man/journald.conf.xml b/man/journald.conf.xml
index 85146b0..52de2f8 100644
--- a/man/journald.conf.xml
+++ b/man/journald.conf.xml
@@ -52,7 +52,7 @@
     <para><filename>/etc/systemd/journald.conf</filename></para>
     <para><filename>/etc/systemd/journald.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/journald.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/journald.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/journald.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/logind.conf.xml b/man/logind.conf.xml
index d02d573..d5fe802 100644
--- a/man/logind.conf.xml
+++ b/man/logind.conf.xml
@@ -52,7 +52,7 @@
     <para><filename>/etc/systemd/logind.conf</filename></para>
     <para><filename>/etc/systemd/logind.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/logind.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/logind.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/logind.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/resolved.conf.xml b/man/resolved.conf.xml
index 8047a4e..2f5c06a 100644
--- a/man/resolved.conf.xml
+++ b/man/resolved.conf.xml
@@ -52,7 +52,7 @@
     <para><filename>/etc/systemd/resolved.conf</filename></para>
     <para><filename>/etc/systemd/resolved.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/resolved.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/resolved.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/resolved.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/sysctl.d.xml b/man/sysctl.d.xml
index 8a13179..af26655 100644
--- a/man/sysctl.d.xml
+++ b/man/sysctl.d.xml
@@ -127,7 +127,7 @@
       <para><filename>/etc/udev/rules.d/99-bridge.rules</filename>:
       </para>
 
-      <programlisting>ACTION=="add", SUBSYSTEM=="module", KERNEL=="bridge", RUN+="/usr/lib/systemd/systemd-sysctl --prefix=/net/bridge"
+      <programlisting>ACTION=="add", SUBSYSTEM=="module", KERNEL=="bridge", RUN+="/lib/systemd/systemd-sysctl --prefix=/net/bridge"
 </programlisting>
 
       <para><filename>/etc/sysctl.d/bridge.conf</filename>:
diff --git a/man/systemctl.xml b/man/systemctl.xml
index 50e6bc9..c9c5b78 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -410,8 +410,8 @@
           processes. Not all unit types manage processes of these
           types however. For example, for mount units, control processes
           are defined (which are the invocations of
-          <filename>/usr/bin/mount</filename> and
-          <filename>/usr/bin/umount</filename>), but no main process
+          <filename>/bin/mount</filename> and
+          <filename>/bin/umount</filename>), but no main process
           is defined. If omitted, defaults to
           <option>all</option>.</para>
         </listitem>
diff --git a/man/systemd-activate.xml b/man/systemd-activate.xml
index cb68a79..1468b91 100644
--- a/man/systemd-activate.xml
+++ b/man/systemd-activate.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <cmdsynopsis>
-      <command>/usr/lib/systemd/systemd-activate</command>
+      <command>/lib/systemd/systemd-activate</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
       <arg choice="plain"><replaceable>daemon</replaceable></arg>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
@@ -149,13 +149,13 @@
     <example>
       <title>Run an echo server on port 2000</title>
 
-      <programlisting>$ /usr/lib/systemd/systemd-activate -l 2000 -a cat</programlisting>
+      <programlisting>$ /lib/systemd/systemd-activate -l 2000 -a cat</programlisting>
     </example>
 
     <example>
       <title>Run a socket activated instance of <citerefentry><refentrytitle>systemd-journal-gatewayd</refentrytitle><manvolnum>8</manvolnum></citerefentry></title>
 
-      <programlisting>$ /usr/lib/systemd/systemd-activate -l 19531 /usr/lib/systemd/systemd-journal-gatewayd</programlisting>
+      <programlisting>$ /lib/systemd/systemd-activate -l 19531 /lib/systemd/systemd-journal-gatewayd</programlisting>
     </example>
   </refsect1>
 
diff --git a/man/systemd-backlight@.service.xml b/man/systemd-backlight@.service.xml
index a259f5d..aa86e8a 100644
--- a/man/systemd-backlight@.service.xml
+++ b/man/systemd-backlight@.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-backlight@.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-backlight</filename></para>
+    <para><filename>/lib/systemd/systemd-backlight</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-binfmt.service.xml b/man/systemd-binfmt.service.xml
index 66d2643..e4d3c56 100644
--- a/man/systemd-binfmt.service.xml
+++ b/man/systemd-binfmt.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-binfmt.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-binfmt</filename></para>
+    <para><filename>/lib/systemd/systemd-binfmt</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-bootchart.xml b/man/systemd-bootchart.xml
index 5386667..dac7eb9 100644
--- a/man/systemd-bootchart.xml
+++ b/man/systemd-bootchart.xml
@@ -60,7 +60,7 @@
       usage, as well as per-process information from a running system.
       Collected results are output as an SVG graph. Normally,
       systemd-bootchart is invoked by the kernel by passing
-      <option>init=<filename>/usr/lib/systemd/systemd-bootchart</filename></option>
+      <option>init=<filename>/lib/systemd/systemd-bootchart</filename></option>
       on the kernel command line. systemd-bootchart will then fork the
       real init off to resume normal system startup, while monitoring
       and logging startup information in the background.
@@ -106,7 +106,7 @@
         <listitem><para>The kernel can invoke
         <command>systemd-bootchart</command> instead of the init
         process. In turn, <command>systemd-bootchart</command> will
-        invoke <command>/usr/lib/systemd/systemd</command>.
+        invoke <command>/lib/systemd/systemd</command>.
         </para></listitem>
       </varlistentry>
 
@@ -201,7 +201,7 @@
         <term><option>-i</option></term>
         <term><option>--init <replaceable>path</replaceable></option></term>
         <listitem><para>Use this init binary. Defaults to
-        <command>/usr/lib/systemd/systemd</command>.
+        <command>/lib/systemd/systemd</command>.
         </para></listitem>
       </varlistentry>
 
diff --git a/man/systemd-bus-proxyd.xml b/man/systemd-bus-proxyd.xml
index e0efe99..69f375d 100644
--- a/man/systemd-bus-proxyd.xml
+++ b/man/systemd-bus-proxyd.xml
@@ -50,7 +50,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
   <refsynopsisdiv>
     <cmdsynopsis>
-      <command>/usr/lib/systemd/systemd-bus-proxyd</command>
+      <command>/lib/systemd/systemd-bus-proxyd</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
       <arg choice="opt"><replaceable>PLACEHOLDER</replaceable></arg>
     </cmdsynopsis>
diff --git a/man/systemd-coredump.xml b/man/systemd-coredump.xml
index 6a7ced1..e8d156c 100644
--- a/man/systemd-coredump.xml
+++ b/man/systemd-coredump.xml
@@ -49,7 +49,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/systemd-coredump</filename></para>
+    <para><filename>/lib/systemd/systemd-coredump</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-cryptsetup-generator.xml b/man/systemd-cryptsetup-generator.xml
index b627035..4c0a8fa 100644
--- a/man/systemd-cryptsetup-generator.xml
+++ b/man/systemd-cryptsetup-generator.xml
@@ -46,7 +46,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-cryptsetup-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-cryptsetup-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-cryptsetup@.service.xml b/man/systemd-cryptsetup@.service.xml
index ea52485..0280c0b 100644
--- a/man/systemd-cryptsetup@.service.xml
+++ b/man/systemd-cryptsetup@.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-cryptsetup@.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-cryptsetup</filename></para>
+    <para><filename>/lib/systemd/systemd-cryptsetup</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-debug-generator.xml b/man/systemd-debug-generator.xml
index 5c5e9fc..3a5ef26 100644
--- a/man/systemd-debug-generator.xml
+++ b/man/systemd-debug-generator.xml
@@ -47,7 +47,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-debug-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-debug-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-efi-boot-generator.xml b/man/systemd-efi-boot-generator.xml
index 3431c3c..9b6f1e1 100644
--- a/man/systemd-efi-boot-generator.xml
+++ b/man/systemd-efi-boot-generator.xml
@@ -48,7 +48,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-efi-boot-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-efi-boot-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-fsck@.service.xml b/man/systemd-fsck@.service.xml
index 69b8fdd..ea25a60 100644
--- a/man/systemd-fsck@.service.xml
+++ b/man/systemd-fsck@.service.xml
@@ -50,7 +50,7 @@
   <refsynopsisdiv>
     <para><filename>systemd-fsck@.service</filename></para>
     <para><filename>systemd-fsck-root.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-fsck</filename></para>
+    <para><filename>/lib/systemd/systemd-fsck</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -137,5 +137,4 @@
       <citerefentry project='man-pages'><refentrytitle>fsck.xfs</refentrytitle><manvolnum>8</manvolnum></citerefentry>
     </para>
   </refsect1>
-
 </refentry>
diff --git a/man/systemd-fstab-generator.xml b/man/systemd-fstab-generator.xml
index bdc2dc1..0f32e70 100644
--- a/man/systemd-fstab-generator.xml
+++ b/man/systemd-fstab-generator.xml
@@ -46,7 +46,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-fstab-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-fstab-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-getty-generator.xml b/man/systemd-getty-generator.xml
index 3389259..0429e59 100644
--- a/man/systemd-getty-generator.xml
+++ b/man/systemd-getty-generator.xml
@@ -47,7 +47,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-getty-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-getty-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-gpt-auto-generator.xml b/man/systemd-gpt-auto-generator.xml
index bcc64ec..98b8f09 100644
--- a/man/systemd-gpt-auto-generator.xml
+++ b/man/systemd-gpt-auto-generator.xml
@@ -50,7 +50,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-gpt-auto-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-gpt-auto-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-halt.service.xml b/man/systemd-halt.service.xml
index c94e2a1..9be1832 100644
--- a/man/systemd-halt.service.xml
+++ b/man/systemd-halt.service.xml
@@ -56,7 +56,7 @@
     <para><filename>systemd-poweroff.service</filename></para>
     <para><filename>systemd-reboot.service</filename></para>
     <para><filename>systemd-kexec.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-shutdown</filename></para>
+    <para><filename>/lib/systemd/systemd-shutdown</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -75,7 +75,7 @@
 
     <para>When these services are run, they ensure that PID 1 is
     replaced by the
-    <filename>/usr/lib/systemd/systemd-shutdown</filename> tool which
+    <filename>/lib/systemd/systemd-shutdown</filename> tool which
     is then responsible for the actual shutdown. Before shutting down,
     this binary will try to unmount all remaining file systems,
     disable all remaining swap devices, detach all remaining storage
@@ -90,7 +90,7 @@
     <para>Immediately before executing the actual system
     halt/poweroff/reboot/kexec <filename>systemd-shutdown</filename>
     will run all executables in
-    <filename>/usr/lib/systemd/system-shutdown/</filename> and pass
+    <filename>/lib/systemd/system-shutdown/</filename> and pass
     one arguments to them: either <literal>halt</literal>,
     <literal>poweroff</literal>, <literal>reboot</literal> or
     <literal>kexec</literal>, depending on the chosen action. All
diff --git a/man/systemd-hibernate-resume-generator.xml b/man/systemd-hibernate-resume-generator.xml
index d811b9b..51e84ef 100644
--- a/man/systemd-hibernate-resume-generator.xml
+++ b/man/systemd-hibernate-resume-generator.xml
@@ -46,7 +46,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-hibernate-resume-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-hibernate-resume-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-hibernate-resume@.service.xml b/man/systemd-hibernate-resume@.service.xml
index 7d00827..ffb579e 100644
--- a/man/systemd-hibernate-resume@.service.xml
+++ b/man/systemd-hibernate-resume@.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-hibernate-resume@.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-hibernate-resume</filename></para>
+    <para><filename>/lib/systemd/systemd-hibernate-resume</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-hostnamed.service.xml b/man/systemd-hostnamed.service.xml
index 6990d41..0b817bf 100644
--- a/man/systemd-hostnamed.service.xml
+++ b/man/systemd-hostnamed.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-hostnamed.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-hostnamed</filename></para>
+    <para><filename>/lib/systemd/systemd-hostnamed</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-hwdb.xml b/man/systemd-hwdb.xml
index f1a1402..68bfdb0 100644
--- a/man/systemd-hwdb.xml
+++ b/man/systemd-hwdb.xml
@@ -57,7 +57,7 @@
       <varlistentry>
         <term><option>--usr</option></term>
         <listitem>
-          <para>Generate in /usr/lib/udev instead of /etc/udev.</para>
+          <para>Generate in /lib/udev instead of /etc/udev.</para>
         </listitem>
       </varlistentry>
       <varlistentry>
diff --git a/man/systemd-initctl.service.xml b/man/systemd-initctl.service.xml
index 5c7f9a4..86db1c1 100644
--- a/man/systemd-initctl.service.xml
+++ b/man/systemd-initctl.service.xml
@@ -52,7 +52,7 @@
   <refsynopsisdiv>
     <para><filename>systemd-initctl.service</filename></para>
     <para><filename>systemd-initctl.socket</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-initctl</filename></para>
+    <para><filename>/lib/systemd/systemd-initctl</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-journal-gatewayd.service.xml b/man/systemd-journal-gatewayd.service.xml
index 7420b59..e80c978 100644
--- a/man/systemd-journal-gatewayd.service.xml
+++ b/man/systemd-journal-gatewayd.service.xml
@@ -54,7 +54,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <para><filename>systemd-journal-gatewayd.service</filename></para>
     <para><filename>systemd-journal-gatewayd.socket</filename></para>
     <cmdsynopsis>
-      <command>/usr/lib/systemd/systemd-journal-gatewayd</command>
+      <command>/lib/systemd/systemd-journal-gatewayd</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
     </cmdsynopsis>
   </refsynopsisdiv>
diff --git a/man/systemd-journald.service.xml b/man/systemd-journald.service.xml
index 8280d6c..f8f7511 100644
--- a/man/systemd-journald.service.xml
+++ b/man/systemd-journald.service.xml
@@ -54,7 +54,7 @@
     <para><filename>systemd-journald.service</filename></para>
     <para><filename>systemd-journald.socket</filename></para>
     <para><filename>systemd-journald-dev-log.socket</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-journald</filename></para>
+    <para><filename>/lib/systemd/systemd-journald</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-localed.service.xml b/man/systemd-localed.service.xml
index 06aa78c..1f258bc 100644
--- a/man/systemd-localed.service.xml
+++ b/man/systemd-localed.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-localed.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-localed</filename></para>
+    <para><filename>/lib/systemd/systemd-localed</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-logind.service.xml b/man/systemd-logind.service.xml
index 5733e42..272d168 100644
--- a/man/systemd-logind.service.xml
+++ b/man/systemd-logind.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-logind.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-logind</filename></para>
+    <para><filename>/lib/systemd/systemd-logind</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-machine-id-commit.service.xml b/man/systemd-machine-id-commit.service.xml
index 7c8fc08..aba5dd0 100644
--- a/man/systemd-machine-id-commit.service.xml
+++ b/man/systemd-machine-id-commit.service.xml
@@ -47,7 +47,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-machine-id-commit.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-machine-id-commit</filename></para>
+    <para><filename>/lib/systemd/systemd-machine-id-commit</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-machined.service.xml b/man/systemd-machined.service.xml
index 999aeee..da5a80f 100644
--- a/man/systemd-machined.service.xml
+++ b/man/systemd-machined.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-machined.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-machined</filename></para>
+    <para><filename>/lib/systemd/systemd-machined</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-modules-load.service.xml b/man/systemd-modules-load.service.xml
index dacd083..8bf9325 100644
--- a/man/systemd-modules-load.service.xml
+++ b/man/systemd-modules-load.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-modules-load.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-modules-load</filename></para>
+    <para><filename>/lib/systemd/systemd-modules-load</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-networkd-wait-online.service.xml b/man/systemd-networkd-wait-online.service.xml
index f53b337..944b73f 100644
--- a/man/systemd-networkd-wait-online.service.xml
+++ b/man/systemd-networkd-wait-online.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-networkd-wait-online.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-networkd-wait-online</filename></para>
+    <para><filename>/lib/systemd/systemd-networkd-wait-online</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-networkd.service.xml b/man/systemd-networkd.service.xml
index 0bfe551..a4a1580 100644
--- a/man/systemd-networkd.service.xml
+++ b/man/systemd-networkd.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-networkd.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-networkd</filename></para>
+    <para><filename>/lib/systemd/systemd-networkd</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -75,7 +75,7 @@
 
   <refsect1><title>Configuration Files</title>
     <para>The configuration files are read from the files located in the
-    system network directory <filename>/usr/lib/systemd/network</filename>,
+    system network directory <filename>/lib/systemd/network</filename>,
     the volatile runtime network directory
     <filename>/run/systemd/network</filename> and the local administration
     network directory <filename>/etc/systemd/network</filename>.</para>
diff --git a/man/systemd-quotacheck.service.xml b/man/systemd-quotacheck.service.xml
index 9d49762..78804ee 100644
--- a/man/systemd-quotacheck.service.xml
+++ b/man/systemd-quotacheck.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-quotacheck.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-quotacheck</filename></para>
+    <para><filename>/lib/systemd/systemd-quotacheck</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-random-seed.service.xml b/man/systemd-random-seed.service.xml
index 8c83668..3d259c0 100644
--- a/man/systemd-random-seed.service.xml
+++ b/man/systemd-random-seed.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-random-seed.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-random-seed</filename></para>
+    <para><filename>/lib/systemd/systemd-random-seed</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-remount-fs.service.xml b/man/systemd-remount-fs.service.xml
index 8e60e31..c1ba9ff 100644
--- a/man/systemd-remount-fs.service.xml
+++ b/man/systemd-remount-fs.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-remount-fs.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-remount-fs</filename></para>
+    <para><filename>/lib/systemd/systemd-remount-fs</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-resolved.service.xml b/man/systemd-resolved.service.xml
index 89ec5f8..e60bc35 100644
--- a/man/systemd-resolved.service.xml
+++ b/man/systemd-resolved.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-resolved.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-resolved</filename></para>
+    <para><filename>/lib/systemd/systemd-resolved</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-rfkill@.service.xml b/man/systemd-rfkill@.service.xml
index 709b09d..4f2274b 100644
--- a/man/systemd-rfkill@.service.xml
+++ b/man/systemd-rfkill@.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-rfkill@.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-rfkill</filename></para>
+    <para><filename>/lib/systemd/systemd-rfkill</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-shutdownd.service.xml b/man/systemd-shutdownd.service.xml
index 756949c..0f42798 100644
--- a/man/systemd-shutdownd.service.xml
+++ b/man/systemd-shutdownd.service.xml
@@ -52,7 +52,7 @@
   <refsynopsisdiv>
     <para><filename>systemd-shutdownd.service</filename></para>
     <para><filename>systemd-shutdownd.socket</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-shutdownd</filename></para>
+    <para><filename>/lib/systemd/systemd-shutdownd</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-sleep.conf.xml b/man/systemd-sleep.conf.xml
index bb17ec8..aaa2f5c 100644
--- a/man/systemd-sleep.conf.xml
+++ b/man/systemd-sleep.conf.xml
@@ -52,7 +52,7 @@ along with systemd; If not, see <http://www.gnu.org/licenses/>.
     <para><filename>/etc/systemd/sleep.conf</filename></para>
     <para><filename>/etc/systemd/sleep.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/sleep.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/sleep.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/sleep.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-socket-proxyd.xml b/man/systemd-socket-proxyd.xml
index 0b852e6..7796226 100644
--- a/man/systemd-socket-proxyd.xml
+++ b/man/systemd-socket-proxyd.xml
@@ -113,7 +113,7 @@ Requires=nginx.service
 After=nginx.service
 
 [Service]
-ExecStart=/usr/lib/systemd/systemd-socket-proxyd /tmp/nginx.sock
+ExecStart=/lib/systemd/systemd-socket-proxyd /tmp/nginx.sock
 PrivateTmp=yes
 PrivateNetwork=yes]]></programlisting>
       </example>
@@ -156,7 +156,7 @@ After=nginx.service
 JoinsNamespaceOf=nginx.service
 
 [Service]
-ExecStart=/usr/lib/systemd/systemd-socket-proxyd 127.0.0.1:8080
+ExecStart=/lib/systemd/systemd-socket-proxyd 127.0.0.1:8080
 PrivateTmp=yes
 PrivateNetwork=yes]]></programlisting>
       </example>
diff --git a/man/systemd-suspend.service.xml b/man/systemd-suspend.service.xml
index a8beb86..071ec88 100644
--- a/man/systemd-suspend.service.xml
+++ b/man/systemd-suspend.service.xml
@@ -56,7 +56,7 @@
     <para><filename>systemd-suspend.service</filename></para>
     <para><filename>systemd-hibernate.service</filename></para>
     <para><filename>systemd-hybrid-sleep.service</filename></para>
-    <para><filename>/usr/lib/systemd/system-sleep</filename></para>
+    <para><filename>/lib/systemd/system-sleep</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -75,7 +75,7 @@
     <para>Immediately before entering system suspend and/or
     hibernation <filename>systemd-suspend.service</filename> (and the
     other mentioned units, respectively) will run all executables in
-    <filename>/usr/lib/systemd/system-sleep/</filename> and pass two
+    <filename>/lib/systemd/system-sleep/</filename> and pass two
     arguments to them. The first argument will be
     <literal>pre</literal>, the second either
     <literal>suspend</literal>, <literal>hibernate</literal>, or
@@ -87,7 +87,7 @@
     until all executables have finished.</para>
 
     <para>Note that scripts or binaries dropped in
-    <filename>/usr/lib/systemd/system-sleep/</filename> are intended
+    <filename>/lib/systemd/system-sleep/</filename> are intended
     for local use only and should be considered hacks. If applications
     want to be notified of system suspend/hibernation and resume,
     there are much nicer interfaces available.</para>
diff --git a/man/systemd-sysctl.service.xml b/man/systemd-sysctl.service.xml
index d4c1a7e..062251e 100644
--- a/man/systemd-sysctl.service.xml
+++ b/man/systemd-sysctl.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-sysctl.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-sysctl</filename></para>
+    <para><filename>/lib/systemd/systemd-sysctl</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-system-update-generator.xml b/man/systemd-system-update-generator.xml
index e7fc95c..43ef5f0 100644
--- a/man/systemd-system-update-generator.xml
+++ b/man/systemd-system-update-generator.xml
@@ -46,7 +46,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-system-update-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-system-update-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-system.conf.xml b/man/systemd-system.conf.xml
index 1b74ed3..0adb840 100644
--- a/man/systemd-system.conf.xml
+++ b/man/systemd-system.conf.xml
@@ -54,11 +54,11 @@
     <para><filename>/etc/systemd/system.conf</filename></para>
     <para><filename>/etc/systemd/system.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/system.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/system.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/system.conf.d/*.conf</filename></para>
     <para><filename>/etc/systemd/user.conf</filename></para>
     <para><filename>/etc/systemd/user.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/user.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/user.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/user.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-sysv-generator.xml b/man/systemd-sysv-generator.xml
index f2d56cb..8fde793 100644
--- a/man/systemd-sysv-generator.xml
+++ b/man/systemd-sysv-generator.xml
@@ -46,7 +46,7 @@
   </refnamediv>
 
   <refsynopsisdiv>
-    <para><filename>/usr/lib/systemd/system-generators/systemd-sysv-generator</filename></para>
+    <para><filename>/lib/systemd/system-generators/systemd-sysv-generator</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-timedated.service.xml b/man/systemd-timedated.service.xml
index e44163a..72fcbd4 100644
--- a/man/systemd-timedated.service.xml
+++ b/man/systemd-timedated.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-timedated.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-timedated</filename></para>
+    <para><filename>/lib/systemd/systemd-timedated</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-timesyncd.service.xml b/man/systemd-timesyncd.service.xml
index ac1af2d..85e43be 100644
--- a/man/systemd-timesyncd.service.xml
+++ b/man/systemd-timesyncd.service.xml
@@ -50,7 +50,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-timesyncd.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-timesyncd</filename></para>
+    <para><filename>/lib/systemd/systemd-timesyncd</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -100,5 +100,4 @@
       <citerefentry project='man-pages'><refentrytitle>hwclock</refentrytitle><manvolnum>8</manvolnum></citerefentry>
     </para>
   </refsect1>
-
 </refentry>
diff --git a/man/systemd-udevd.service.xml b/man/systemd-udevd.service.xml
index b18b228..1689754 100644
--- a/man/systemd-udevd.service.xml
+++ b/man/systemd-udevd.service.xml
@@ -37,7 +37,7 @@
     <para><filename>systemd-udevd-kernel.socket</filename></para>
 
     <cmdsynopsis>
-      <command>/usr/lib/systemd/systemd-udevd</command>
+      <command>/lib/systemd/systemd-udevd</command>
       <arg><option>--daemon</option></arg>
       <arg><option>--debug</option></arg>
       <arg><option>--children-max=</option></arg>
diff --git a/man/systemd-update-done.service.xml b/man/systemd-update-done.service.xml
index d65f175..bf40b55 100644
--- a/man/systemd-update-done.service.xml
+++ b/man/systemd-update-done.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-update-done.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-update-done</filename></para>
+    <para><filename>/lib/systemd/systemd-update-done</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-update-utmp.service.xml b/man/systemd-update-utmp.service.xml
index c8a9cb7..0a175c9 100644
--- a/man/systemd-update-utmp.service.xml
+++ b/man/systemd-update-utmp.service.xml
@@ -51,7 +51,7 @@
   <refsynopsisdiv>
     <para><filename>systemd-update-utmp.service</filename></para>
     <para><filename>systemd-update-utmp-runlevel.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-update-utmp</filename></para>
+    <para><filename>/lib/systemd/systemd-update-utmp</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd-user-sessions.service.xml b/man/systemd-user-sessions.service.xml
index e75ef11..225f206 100644
--- a/man/systemd-user-sessions.service.xml
+++ b/man/systemd-user-sessions.service.xml
@@ -48,9 +48,8 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-user-sessions.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-user-sessions</filename></para>
+    <para><filename>/lib/systemd/systemd-user-sessions</filename></para>
   </refsynopsisdiv>
-
   <refsect1>
     <title>Description</title>
 
diff --git a/man/systemd-vconsole-setup.service.xml b/man/systemd-vconsole-setup.service.xml
index 7c6ed08..cf9674e 100644
--- a/man/systemd-vconsole-setup.service.xml
+++ b/man/systemd-vconsole-setup.service.xml
@@ -48,7 +48,7 @@
 
   <refsynopsisdiv>
     <para><filename>systemd-vconsole-setup.service</filename></para>
-    <para><filename>/usr/lib/systemd/systemd-vconsole-setup</filename></para>
+    <para><filename>/lib/systemd/systemd-vconsole-setup</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/systemd.directives.xml b/man/systemd.directives.xml
index c65c0dc..d329492 100644
--- a/man/systemd.directives.xml
+++ b/man/systemd.directives.xml
@@ -3198,9 +3198,9 @@
 
 <varlistentry><term><filename>/usr/bin</filename></term><listitem><para><citerefentry><refentrytitle>file-hierarchy</refentrytitle><manvolnum>7</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/bin/mount</filename></term><listitem><para><citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/bin/mount</filename></term><listitem><para><citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/bin/umount</filename></term><listitem><para><citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/bin/umount</filename></term><listitem><para><citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/include</filename></term><listitem><para><citerefentry><refentrytitle>file-hierarchy</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
@@ -3222,139 +3222,139 @@
 
 <varlistentry><term><filename>/usr/lib/sysctl.d/50-coredump.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-coredump</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/bootchart.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>bootchart.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/bootchart.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>bootchart.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/coredump.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>coredump.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/coredump.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>coredump.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/lib/systemd/import-pubring.gpg</filename></term><listitem><para><citerefentry><refentrytitle>machinectl</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/journald.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/journald.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/logind.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/logind.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>logind.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/network</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.link</refentrytitle><manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.netdev</refentrytitle><manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/network</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.link</refentrytitle><manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.netdev</refentrytitle><manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/resolved.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>resolved.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/resolved.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>resolved.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/sleep.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sleep.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/sleep.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sleep.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system</filename></term><listitem><para><citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system</filename></term><listitem><para><citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/</filename></term><listitem><para><citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/</filename></term><listitem><para><citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-cryptsetup-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-cryptsetup-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-cryptsetup-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-cryptsetup-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-debug-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-debug-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-debug-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-debug-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-efi-boot-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-efi-boot-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-efi-boot-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-efi-boot-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-fstab-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-fstab-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-fstab-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-fstab-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-getty-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-getty-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-getty-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-getty-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-gpt-auto-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-gpt-auto-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-gpt-auto-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-gpt-auto-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-hibernate-resume-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hibernate-resume-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-hibernate-resume-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hibernate-resume-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-system-update-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system-update-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-system-update-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system-update-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-generators/systemd-sysv-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sysv-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-generators/systemd-sysv-generator</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sysv-generator</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-preset/*.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-preset/*.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-preset/50-gnome.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-preset/50-gnome.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-preset/99-default.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-preset/99-default.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-shutdown/</filename></term><listitem><para><citerefentry><refentrytitle>systemd-halt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-shutdown/</filename></term><listitem><para><citerefentry><refentrytitle>systemd-halt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system-sleep</filename></term><listitem><para><citerefentry><refentrytitle>systemd-suspend.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system-sleep</filename></term><listitem><para><citerefentry><refentrytitle>systemd-suspend.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/system/httpd.service</filename></term><listitem><para><citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/system/httpd.service</filename></term><listitem><para><citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd</filename></term><listitem><para><citerefentry><refentrytitle>bootchart.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd</filename></term><listitem><para><citerefentry><refentrytitle>bootchart.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><command>/usr/lib/systemd/systemd-activate</command></term><listitem><para><citerefentry><refentrytitle>systemd-activate</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><command>/lib/systemd/systemd-activate</command></term><listitem><para><citerefentry><refentrytitle>systemd-activate</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-backlight</filename></term><listitem><para><citerefentry><refentrytitle>systemd-backlight@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-backlight</filename></term><listitem><para><citerefentry><refentrytitle>systemd-backlight@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-binfmt</filename></term><listitem><para><citerefentry><refentrytitle>systemd-binfmt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-binfmt</filename></term><listitem><para><citerefentry><refentrytitle>systemd-binfmt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-bootchart</filename></term><listitem><para><citerefentry><refentrytitle>systemd-bootchart</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-bootchart</filename></term><listitem><para><citerefentry><refentrytitle>systemd-bootchart</refentrytitle><manvolnum>1</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><command>/usr/lib/systemd/systemd-bus-proxyd</command></term><listitem><para><citerefentry><refentrytitle>systemd-bus-proxyd</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><command>/lib/systemd/systemd-bus-proxyd</command></term><listitem><para><citerefentry><refentrytitle>systemd-bus-proxyd</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-coredump</filename></term><listitem><para><citerefentry><refentrytitle>systemd-coredump</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-coredump</filename></term><listitem><para><citerefentry><refentrytitle>systemd-coredump</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-cryptsetup</filename></term><listitem><para><citerefentry><refentrytitle>systemd-cryptsetup@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-cryptsetup</filename></term><listitem><para><citerefentry><refentrytitle>systemd-cryptsetup@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-fsck</filename></term><listitem><para><citerefentry><refentrytitle>systemd-fsck@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-fsck</filename></term><listitem><para><citerefentry><refentrytitle>systemd-fsck@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-hibernate-resume</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hibernate-resume@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-hibernate-resume</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hibernate-resume@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-hostnamed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hostnamed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-hostnamed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-hostnamed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-initctl</filename></term><listitem><para><citerefentry><refentrytitle>systemd-initctl.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-initctl</filename></term><listitem><para><citerefentry><refentrytitle>systemd-initctl.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><command>/usr/lib/systemd/systemd-journal-gatewayd</command></term><listitem><para><citerefentry><refentrytitle>systemd-journal-gatewayd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><command>/lib/systemd/systemd-journal-gatewayd</command></term><listitem><para><citerefentry><refentrytitle>systemd-journal-gatewayd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-journald</filename></term><listitem><para><citerefentry><refentrytitle>systemd-journald.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-journald</filename></term><listitem><para><citerefentry><refentrytitle>systemd-journald.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-localed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-localed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-localed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-localed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-logind</filename></term><listitem><para><citerefentry><refentrytitle>systemd-logind.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-logind</filename></term><listitem><para><citerefentry><refentrytitle>systemd-logind.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-machine-id-commit</filename></term><listitem><para><citerefentry><refentrytitle>systemd-machine-id-commit.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-machine-id-commit</filename></term><listitem><para><citerefentry><refentrytitle>systemd-machine-id-commit.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-machined</filename></term><listitem><para><citerefentry><refentrytitle>systemd-machined.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-machined</filename></term><listitem><para><citerefentry><refentrytitle>systemd-machined.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-modules-load</filename></term><listitem><para><citerefentry><refentrytitle>systemd-modules-load.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-modules-load</filename></term><listitem><para><citerefentry><refentrytitle>systemd-modules-load.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-networkd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-networkd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-networkd-wait-online</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd-wait-online.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-networkd-wait-online</filename></term><listitem><para><citerefentry><refentrytitle>systemd-networkd-wait-online.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-quotacheck</filename></term><listitem><para><citerefentry><refentrytitle>systemd-quotacheck.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-quotacheck</filename></term><listitem><para><citerefentry><refentrytitle>systemd-quotacheck.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-random-seed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-random-seed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-random-seed</filename></term><listitem><para><citerefentry><refentrytitle>systemd-random-seed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-remount-fs</filename></term><listitem><para><citerefentry><refentrytitle>systemd-remount-fs.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-remount-fs</filename></term><listitem><para><citerefentry><refentrytitle>systemd-remount-fs.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-resolved</filename></term><listitem><para><citerefentry><refentrytitle>systemd-resolved.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-resolved</filename></term><listitem><para><citerefentry><refentrytitle>systemd-resolved.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-rfkill</filename></term><listitem><para><citerefentry><refentrytitle>systemd-rfkill@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-rfkill</filename></term><listitem><para><citerefentry><refentrytitle>systemd-rfkill@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-shutdown</filename></term><listitem><para><citerefentry><refentrytitle>systemd-halt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-shutdown</filename></term><listitem><para><citerefentry><refentrytitle>systemd-halt.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-shutdownd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-shutdownd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-shutdownd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-shutdownd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-sysctl</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sysctl.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-sysctl</filename></term><listitem><para><citerefentry><refentrytitle>systemd-sysctl.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-timedated</filename></term><listitem><para><citerefentry><refentrytitle>systemd-timedated.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-timedated</filename></term><listitem><para><citerefentry><refentrytitle>systemd-timedated.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-timesyncd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-timesyncd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-timesyncd</filename></term><listitem><para><citerefentry><refentrytitle>systemd-timesyncd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><command>/usr/lib/systemd/systemd-udevd</command></term><listitem><para><citerefentry><refentrytitle>systemd-udevd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><command>/lib/systemd/systemd-udevd</command></term><listitem><para><citerefentry><refentrytitle>systemd-udevd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-update-done</filename></term><listitem><para><citerefentry><refentrytitle>systemd-update-done.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-update-done</filename></term><listitem><para><citerefentry><refentrytitle>systemd-update-done.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-update-utmp</filename></term><listitem><para><citerefentry><refentrytitle>systemd-update-utmp.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-update-utmp</filename></term><listitem><para><citerefentry><refentrytitle>systemd-update-utmp.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-user-sessions</filename></term><listitem><para><citerefentry><refentrytitle>systemd-user-sessions.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-user-sessions</filename></term><listitem><para><citerefentry><refentrytitle>systemd-user-sessions.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/systemd-vconsole-setup</filename></term><listitem><para><citerefentry><refentrytitle>systemd-vconsole-setup.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/systemd-vconsole-setup</filename></term><listitem><para><citerefentry><refentrytitle>systemd-vconsole-setup.service</refentrytitle><manvolnum>8</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/timesyncd.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>timesyncd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/timesyncd.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>timesyncd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/user/</filename></term><listitem><para><citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/user/</filename></term><listitem><para><citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/lib/systemd/user-generators/</filename></term><listitem><para><citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/lib/systemd/user-preset/*.preset</filename></term><listitem><para><citerefentry><refentrytitle>systemd.preset</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/systemd/user.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/systemd/user.conf.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/lib/sysusers.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>sysusers.d</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
@@ -3362,13 +3362,13 @@
 
 <varlistentry><term><filename>/usr/lib/tmpfiles.d/*.conf</filename></term><listitem><para><citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/udev</filename></term><listitem><para><citerefentry><refentrytitle>udev</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/udev</filename></term><listitem><para><citerefentry><refentrytitle>udev</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/udev/hwdb.bin</filename></term><listitem><para><citerefentry><refentrytitle>hwdb</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/udev/hwdb.bin</filename></term><listitem><para><citerefentry><refentrytitle>hwdb</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/udev/hwdb.d</filename></term><listitem><para><citerefentry><refentrytitle>hwdb</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/udev/hwdb.d</filename></term><listitem><para><citerefentry><refentrytitle>hwdb</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
-<varlistentry><term><filename>/usr/lib/udev/rules.d</filename></term><listitem><para><citerefentry><refentrytitle>udev</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
+<varlistentry><term><filename>/lib/udev/rules.d</filename></term><listitem><para><citerefentry><refentrytitle>udev</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
 <varlistentry><term><filename>/usr/lib64</filename></term><listitem><para><citerefentry><refentrytitle>file-hierarchy</refentrytitle><manvolnum>7</manvolnum></citerefentry></para></listitem></varlistentry>
 
diff --git a/man/systemd.link.xml b/man/systemd.link.xml
index 3fac760..116f965 100644
--- a/man/systemd.link.xml
+++ b/man/systemd.link.xml
@@ -56,7 +56,7 @@
     <command>net_setup_link</command> udev builtin.</para>
 
     <para>The link files are read from the files located in the system
-    network directory <filename>/usr/lib/systemd/network</filename>,
+    network directory <filename>/lib/systemd/network</filename>,
     the volatile runtime network directory
     <filename>/run/systemd/network</filename>, and the local
     administration network directory
@@ -67,10 +67,10 @@
     However, files with identical filenames replace each other. Files
     in <filename>/etc</filename> have the highest priority, files in
     <filename>/run</filename> take precedence over files with the same
-    name in <filename>/usr/lib</filename>. This can be used to
+    name in <filename>/lib</filename>. This can be used to
     override a system-supplied link file with a local file if needed;
     a symlink in <filename>/etc</filename> with the same name as a
-    link file in <filename>/usr/lib</filename>, pointing to
+    link file in <filename>/lib</filename>, pointing to
     <filename>/dev/null</filename>, disables the link file
     entirely.</para>
 
diff --git a/man/systemd.netdev.xml b/man/systemd.netdev.xml
index ef58887..a4857a6 100644
--- a/man/systemd.netdev.xml
+++ b/man/systemd.netdev.xml
@@ -68,7 +68,7 @@
 
     <para>The <filename>.netdev</filename> files are read from the
     files located in the system network directory
-    <filename>/usr/lib/systemd/network</filename>, the volatile
+    <filename>/lib/systemd/network</filename>, the volatile
     runtime network directory
     <filename>/run/systemd/network</filename> and the local
     administration network directory
@@ -78,10 +78,10 @@
     identical filenames replace each other. Files in
     <filename>/etc</filename> have the highest priority, files in
     <filename>/run</filename> take precedence over files with the same
-    name in <filename>/usr/lib</filename>. This can be used to
+    name in <filename>/lib</filename>. This can be used to
     override a system-supplied configuration file with a local file if
     needed; a symlink in <filename>/etc</filename> with the same name
-    as a configuration file in <filename>/usr/lib</filename>, pointing
+    as a configuration file in <filename>/lib</filename>, pointing
     to <filename>/dev/null</filename>, disables the configuration file
     entirely.</para>
 
diff --git a/man/systemd.network.xml b/man/systemd.network.xml
index 9738627..2b06cb3 100644
--- a/man/systemd.network.xml
+++ b/man/systemd.network.xml
@@ -64,7 +64,7 @@
 
     <para>The <filename>.network</filename> files are read from the
     files located in the system network directory
-    <filename>/usr/lib/systemd/network</filename>, the volatile
+    <filename>/lib/systemd/network</filename>, the volatile
     runtime network directory
     <filename>/run/systemd/network</filename> and the local
     administration network directory
@@ -74,10 +74,10 @@
     identical filenames replace each other. Files in
     <filename>/etc</filename> have the highest priority, files in
     <filename>/run</filename> take precedence over files with the same
-    name in <filename>/usr/lib</filename>. This can be used to
+    name in <filename>/lib</filename>. This can be used to
     override a system-supplied configuration file with a local file if
     needed; a symlink in <filename>/etc</filename> with the same name
-    as a configuration file in <filename>/usr/lib</filename>, pointing
+    as a configuration file in <filename>/lib</filename>, pointing
     to <filename>/dev/null</filename>, disables the configuration file
     entirely.</para>
 
diff --git a/man/systemd.preset.xml b/man/systemd.preset.xml
index 2f9add8..a30cc5b 100644
--- a/man/systemd.preset.xml
+++ b/man/systemd.preset.xml
@@ -48,10 +48,10 @@
   <refsynopsisdiv>
     <para><filename>/etc/systemd/system-preset/*.preset</filename></para>
     <para><filename>/run/systemd/system-preset/*.preset</filename></para>
-    <para><filename>/usr/lib/systemd/system-preset/*.preset</filename></para>
+    <para><filename>/lib/systemd/system-preset/*.preset</filename></para>
     <para><filename>/etc/systemd/user-preset/*.preset</filename></para>
     <para><filename>/run/systemd/user-preset/*.preset</filename></para>
-    <para><filename>/usr/lib/systemd/user-preset/*.preset</filename></para>
+    <para><filename>/lib/systemd/user-preset/*.preset</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
@@ -108,10 +108,10 @@
     <para>Each preset file shall be named in the style of
     <filename>&lt;priority&gt;-&lt;program&gt;.conf</filename>. Files
     in <filename>/etc/</filename> override files with the same name in
-    <filename>/usr/lib/</filename> and <filename>/run/</filename>.
+    <filename>/lib/</filename> and <filename>/run/</filename>.
     Files in <filename>/run/</filename> override files with the same
-    name in <filename>/usr/lib/</filename>. Packages should install
-    their preset files in <filename>/usr/lib/</filename>. Files in
+    name in <filename>/lib/</filename>. Packages should install
+    their preset files in <filename>/lib/</filename>. Files in
     <filename>/etc/</filename> are reserved for the local
     administrator, who may use this logic to override the preset files
     installed by vendor packages. All preset files are sorted by their
@@ -133,7 +133,7 @@
     <title>Example</title>
 
     <example>
-      <title>Default off example <filename>/usr/lib/systemd/system-preset/99-default.preset</filename>:</title>
+      <title>Default off example <filename>/lib/systemd/system-preset/99-default.preset</filename>:</title>
 
       <programlisting>disable *</programlisting>
     </example>
@@ -144,7 +144,7 @@
     suchlike.</para>
 
     <example>
-      <title>A GNOME spin example <filename>/usr/lib/systemd/system-preset/50-gnome.preset</filename>:</title>
+      <title>A GNOME spin example <filename>/lib/systemd/system-preset/50-gnome.preset</filename>:</title>
 
       <programlisting>enable gdm.service
 enable colord.service
diff --git a/man/systemd.unit.xml b/man/systemd.unit.xml
index c2e374a..9ce8b8b 100644
--- a/man/systemd.unit.xml
+++ b/man/systemd.unit.xml
@@ -66,7 +66,7 @@
 
     <para><literallayout><filename>/etc/systemd/system/*</filename>
 <filename>/run/systemd/system/*</filename>
-<filename>/usr/lib/systemd/system/*</filename>
+<filename>/lib/systemd/system/*</filename>
 <filename>...</filename>
     </literallayout></para>
 
@@ -77,7 +77,7 @@
 <filename>/run/systemd/user/*</filename>
 <filename>$XDG_DATA_HOME/systemd/user/*</filename>
 <filename>$HOME/.local/share/systemd/user/*</filename>
-<filename>/usr/lib/systemd/user/*</filename>
+<filename>/lib/systemd/user/*</filename>
 <filename>...</filename>
     </literallayout></para>
   </refsynopsisdiv>
@@ -295,7 +295,7 @@
       <entry>Runtime units</entry>
           </row>
           <row>
-      <entry><filename>/usr/lib/systemd/system</filename></entry>
+      <entry><filename>/lib/systemd/system</filename></entry>
       <entry>Units of installed packages</entry>
           </row>
         </tbody>
@@ -1309,7 +1309,7 @@ ExecStart=/usr/sbin/foo-daemon
 
       <para>There are two methods of overriding vendor settings in
       unit files: copying the unit file from
-      <filename>/usr/lib/systemd/system</filename> to
+      <filename>/lib/systemd/system</filename> to
       <filename>/etc/systemd/system</filename> and modifying the
       chosen settings. Alternatively, one can create a directory named
       <filename><replaceable>unit</replaceable>.d/</filename> within
@@ -1343,7 +1343,7 @@ ExecStart=/usr/sbin/foo-daemon
       load paths for further details.</para>
 
       <para>Suppose there is a vendor-supplied unit
-      <filename>/usr/lib/systemd/system/httpd.service</filename> with
+      <filename>/lib/systemd/system/httpd.service</filename> with
       the following contents:</para>
 
       <programlisting>[Unit]
diff --git a/man/systemd.xml b/man/systemd.xml
index d006b0b..192a1ec 100644
--- a/man/systemd.xml
+++ b/man/systemd.xml
@@ -438,7 +438,7 @@
         by <command>pkg-config systemd
         --variable=systemdsystemunitdir</command>. Other directories
         checked are <filename>/usr/local/lib/systemd/system</filename>
-        and <filename>/usr/lib/systemd/system</filename>. User
+        and <filename>/lib/systemd/system</filename>. User
         configuration always takes precedence. <command>pkg-config
         systemd --variable=systemdsystemconfdir</command> returns the
         path of the system configuration directory. Packages should
@@ -1091,5 +1091,4 @@
       <citerefentry><refentrytitle>systemd.directives</refentrytitle><manvolnum>7</manvolnum></citerefentry>
     </para>
   </refsect1>
-
 </refentry>
diff --git a/man/timedatectl.xml b/man/timedatectl.xml
index 98ec013..9b777b2 100644
--- a/man/timedatectl.xml
+++ b/man/timedatectl.xml
@@ -210,7 +210,7 @@ Password: ********
 
     <programlisting>$ systemctl status chronyd.service
 chronyd.service - NTP client/server
-    Loaded: loaded (/usr/lib/systemd/system/chronyd.service; enabled)
+    Loaded: loaded (/lib/systemd/system/chronyd.service; enabled)
     Active: active (running) since Fri, 2012-11-02 09:36:25 CET; 5s ago
 ...</programlisting>
     </para>
diff --git a/man/timesyncd.conf.xml b/man/timesyncd.conf.xml
index 89a651c..9cce88b 100644
--- a/man/timesyncd.conf.xml
+++ b/man/timesyncd.conf.xml
@@ -52,7 +52,7 @@
     <para><filename>/etc/systemd/timesyncd.conf</filename></para>
     <para><filename>/etc/systemd/timesyncd.conf.d/*.conf</filename></para>
     <para><filename>/run/systemd/timesyncd.conf.d/*.conf</filename></para>
-    <para><filename>/usr/lib/systemd/timesyncd.conf.d/*.conf</filename></para>
+    <para><filename>/lib/systemd/timesyncd.conf.d/*.conf</filename></para>
   </refsynopsisdiv>
 
   <refsect1>
diff --git a/man/udev.xml b/man/udev.xml
index a948ea7..d3cc373 100644
--- a/man/udev.xml
+++ b/man/udev.xml
@@ -55,17 +55,17 @@
 
   <refsect1><title>Rules Files</title>
       <para>The udev rules are read from the files located in the
-      system rules directory <filename>/usr/lib/udev/rules.d</filename>,
+      system rules directory <filename>/lib/udev/rules.d</filename>,
       the volatile runtime directory <filename>/run/udev/rules.d</filename>
       and the local administration directory <filename>/etc/udev/rules.d</filename>.
       All rules files are collectively sorted and processed in lexical order,
       regardless of the directories in which they live. However, files with
       identical filenames replace each other. Files in <filename>/etc</filename>
       have the highest priority, files in <filename>/run</filename> take precedence
-      over files with the same name in <filename>/usr/lib</filename>. This can be
+      over files with the same name in <filename>/lib</filename>. This can be
       used to override a system-supplied rules file with a local file if needed;
       a symlink in <filename>/etc</filename> with the same name as a rules file in
-      <filename>/usr/lib</filename>, pointing to <filename>/dev/null</filename>,
+      <filename>/lib</filename>, pointing to <filename>/dev/null</filename>,
       disables the rules file entirely. Rule files must have the extension
       <filename>.rules</filename>; other extensions are ignored.</para>
 
@@ -409,7 +409,7 @@
                 <listitem>
                   <para>Execute an external program specified as the assigned
                   value. If no absolute path is given, the program is expected
-                  to live in <filename>/usr/lib/udev</filename>; otherwise, the
+                  to live in <filename>/lib/udev</filename>; otherwise, the
                   absolute path must be specified.</para>
                   <para>This is the default if no <replaceable>type</replaceable>
                   is specified.</para>
